R/tidy-upkeep.R

```diff
@@ -111,6 +111,42 @@ if (year <= 2022) {
     )
   }
 
+  if (year <= 2023) {
+
+    desc <- proj_desc()
+
+    bullets <- c(bullets,
+     "2023",
+     "",
+     "Posit updates:",
+     "",
+     todo('Update copyright holder in DESCRIPTION: `person(given = "Posit, PBC", role = c("cph", "fnd"))`',
+          rstudio_pkg && !rstudio_person_ok),
+     todo("Double check license file uses '[package] authors' as copyright holder. Run `use_mit_license()`",
+          grepl("MIT", desc$get_field("License"))),
```

Comment:
That check is for an MIT license, which I don't think is what the bullet is getting at.  I suspect you can just drop this one.

Reply:
I thought this was for MIT-licensed packages, as that is the pattern in the LICENSE file - based on [this issue](https://github.com/r-lib/usethis/issues/1683) and [Jenny's comment here](https://github.com/r-lib/usethis/pull/1780#discussion_r1123948264)

Reply:
Ah, I get it now.